sdk/core/azure-core/azure/core/messaging.py

```diff
@@ -0,0 +1,157 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import uuid
+from  base64 import b64decode
+from datetime import datetime
+
+try:
+    from datetime import timezone
+    TZ_UTC = timezone.utc  # type: ignore
+except ImportError:
+    from azure.core._utils import _FixedOffset
+    TZ_UTC = _FixedOffset(0) # type: ignore
```

Comment:
Please make it to _utils, and use it if parsing show `Z` , which will make the CloudEvent roudtrippable. Otherwise you send TZ_UTC class, but re-parsing back will give a FixedOffset(0) instance.